prime-dotnet-webapi/Services/SubmissionRulesService.cs

```diff
@@ -48,7 +48,7 @@ public async Task<bool> QualifiesForAutomaticAdjudicationAsync(Enrollee enrollee
                 new IdentityAssuranceLevelRule(),
                 new IdentityProviderRule(),
                 new NoAssignedAgreementRule(),
-                new IsPotentialPaperEnrolleeReturnee(_enrolleePaperSubmissionService),
+                new IsPotentialPaperEnrolleeReturnee(_enrolleePaperSubmissionService, _businessEventService),
```

Comment:
You're passing the constructor's positional arguments arguments in the wrong order
**Passing:** `new IsPotentialPaperEnrolleeReturnee(_enrolleePaperSubmissionService, _businessEventService)`
Constructor: `public IsPotentialPaperEnrolleeReturnee(IBusinessEventService businessEventService,       IEnrolleePaperSubmissionService enrolleePaperSubmissionService)`